proto/storage/product_usage.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+option go_package = "storage";
+option java_package = "io.stackrox.proto.storage";
+
+import "google/protobuf/timestamp.proto";
+import "gogoproto/gogo.proto";
+
+package storage;
+
+// SecuredUnits represents a record of an aggregated secured clusters usage
+// metrics. The metrics are aggregated periodically, and put into the database.
+message SecuredUnits {
```

Comment:
I got the answer to my first question. The units can be any data point node, CPU, mem, namespaces, clusters, etc. Can this proto ever represent data per cluster, or would it always be the aggregation of all quantities of all secured clusters?

Reply:
Our current understanding and the requirement is that we aggregate.